src/System Application/App/Edit in Excel/src/EditinExcelImpl.Codeunit.al

```diff
@@ -224,7 +224,9 @@ codeunit 1482 "Edit in Excel Impl."
         CurrentPosition := 1;
 
         while CurrentPosition <= StrLen(ConvertedName) do begin
-            if ConvertedName[CurrentPosition] in ['''', '+'] then begin
+            // Notice in the following line that – (en dash) is not a normal dash (em dash).
+            // We need to handle this here because at least the norwegian translation uses en dash.
+            if ConvertedName[CurrentPosition] in ['''', '+', '–'] then begin
```

Comment:
Can we add a unit test hitting this path?
Nice that you have a comment but just want to make sure this doesn't slip somehow in the future.

Reply:
@encimita added a test now